core/server/src/main/java/alluxio/RpcUtils.java

```diff
@@ -39,6 +39,7 @@
     try {
       return callable.call();
     } catch (AlluxioException e) {
+      LOG.error("Internal Alluxio error when running rpc", e);
```

Comment:
Good point, I think the Alluxio errors should be debug, but the IO errors probably should be visible.